tests/integration/test_integration_all_rust.py

```diff
@@ -223,13 +224,15 @@ def __getattribute__(self, name: str):
             return f
 
     def connect(self, connection_port: int | None = None):
+        """Connect."""
```

Comment:
Not really sure function descriptions like this help.

Maybe
```suggestion
        """Establish a websocket connection to localhost at the provided `connection_port`."""
```